packages/xo-web/src/common/sorted-table/index.js

```diff
@@ -666,10 +676,18 @@ export default class SortedTable extends Component {
                 selectedItemsIds[method](visibleItems[i].id)
               }
             })
-          : selectedItemsIds[method](item.id),
-    })
+          : selectedItemsIds[method](item.id)
+      this._previous = current
+    }
+
+    if (onSelect !== undefined) {
+      onSelect(filter(visibleItems, item => _selectedItemsIds.has(item.id)))
```

Comment:
`onSelect(_selectedItemsIds.toArray())`

Reply:
We will need the `items` not the `IDs`  of items